src/amo/components/RatingManager/index.js

```diff
@@ -18,34 +21,52 @@ import {
 import translate from 'core/i18n/translate';
 import log from 'core/logger';
 import DefaultRating from 'ui/components/Rating';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { UserReviewType } from 'amo/actions/reviews';
+import type { SubmitReviewParams } from 'amo/api';
+import type { UrlFormatParams } from 'core/api';
+import type { ApiStateType } from 'core/reducers/api';
+import type { DispatchFn } from 'core/types/reduxTypes';
+import type { AddonType, AddonTypeProp, AddonVersionType }
+  from 'core/types/addonTypes';
 
 import './styles.scss';
 
+type LoadSavedReviewFn = ({|
+  userId: number,
+  addonId: number,
+|}) => Promise<*>;
```

Comment:
This is a [function type](https://flow.org/en/docs/types/functions/#toc-function-types). The return value is a `Promise` which is defined as a [generic](https://flow.org/en/docs/types/generics/) since a promise resolve to a specific value. In this case, I'm using `*` to indicate that the promise can resolve to any value. I replaced this with the literal string `any` which isn't exactly the same but it's easier to understand.